optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/BenchmarkResultIO.java

```diff
@@ -108,9 +99,6 @@ protected PlannerBenchmarkResult readPlannerBenchmarkResult(File plannerBenchmar
             String benchmarkReportDirectoryName = plannerBenchmarkResultFile.getParentFile().getName();
             plannerBenchmarkResult = PlannerBenchmarkResult.createUnmarshallingFailedResult(
                     benchmarkReportDirectoryName);
-        } catch (XStreamException e) {
```

Comment:
I am not seeing that with my version of xstream:

    XStreamException extends BaseException
    BaseException extends RuntimeException